llvm/test/CodeGen/CHERI-Generic/regenerate-all.py

```diff
@@ -81,6 +81,9 @@ def __init__(self):
         parser = argparse.ArgumentParser()
         parser.add_argument("--llvm-bindir", type=Path, required=True)
         parser.add_argument("--verbose", action="store_true")
+        parser.add_argument("--arch", default='all', 
```

Comment:
This does have its uses, but you need to be careful; i.e. this is for when you've changed things that only affect a subset of backends, not for when you've changed things that affect all backends but you're just not building all of them. Otherwise they will break. And I suspect for what you're doing you fall into the latter camp?

Reply:
I agree. Most changes will probably have to be propagated to both backends. I've removed this change from this PR, perhaps I'll create a separate patch and discuss it there if we want to include it.